src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -153,6 +157,9 @@ export const BaseArtworkFilter: React.FC<
   const { filtered_artworks } = viewer
   const hasFilter = filtered_artworks && filtered_artworks.id
 
+  const isCreateAlertEnabled = sd.ENABLE_SAVED_SEARCH && enableCreateAlert
+  const showCreateAlert = isCreateAlertEnabled && filterContext.hasFilters
```

Comment:
**question**: Should we consolidate this logic into the `ArtistArtworkFilter.tsx` component instead of updating this component to be aware of the feature flag?

The addition of the component prop here makes sense but I think deciding whether we enable/disable that option should be a bit further up the stack, closer to the artist page's artwork grid, especially since that will be our initial focus as we prep to expand to other surfaces.

Reply:
That makes sense 👍